src/bootstrap.php

```diff
@@ -839,7 +839,7 @@ function (Accept $type, string $id) use ($app) {
         );
     }
 )->before($app['negotiate.accept'](
-    'application/vnd.elife.collection+json; version=1'
+    'application/vnd.elife.collection+json; version=2'
```

Comment:
We'll need to support the old version and filter out the entries, otherwise things will break.